pkg/inspecttypes/dockercompat/dockercompat.go

```diff
@@ -38,7 +38,9 @@ import (
 	"github.com/containerd/nerdctl/pkg/imgutil"
 	"github.com/containerd/nerdctl/pkg/inspecttypes/native"
 	"github.com/containerd/nerdctl/pkg/labels"
+	"github.com/docker/docker/api/types/mount"
 	"github.com/docker/go-connections/nat"
+	"github.com/moby/moby/api/types"
```

Comment:
We do not import this package. Please see other struct definitions and keep the coding style consistent.

Reply:
Okay.

Reply:
What if I used this: https://github.com/moby/moby/blob/master/api/types/types.go#L513